data/data/agent/systemd/units/agent.service.template

```diff
@@ -13,8 +13,11 @@ Environment=no_proxy=
 # https://github.com/openshift/assisted-service/blob/master/internal/ignition/ignition.go#L1381
 Environment=PULL_SECRET_TOKEN={{.PullSecretToken}}
 TimeoutStartSec=3000
-ExecStartPre=podman run --privileged --rm -v /usr/local/bin:/hostbin quay.io/edge-infrastructure/assisted-installer-agent:latest cp /usr/bin/agent /hostbin
-ExecStart=/usr/local/bin/agent --url '{{.ServiceBaseURL}}' --infra-env-id '{{.InfraEnvID}}' --agent-version quay.io/edge-infrastructure/assisted-installer-agent:latest --insecure=true
+ExecStartPre=/usr/local/bin/release-image.sh %t agent-installer-node-agent
+# Read the image from environment file in scripts
+ExecStartPre=/usr/local/bin/pre-start-agent.sh %t %n agent-installer-node-agent
```

Comment:
Maybe we could call this extract-agent or something.

Reply:
Changed name